jira7/jira_datamappers.bal

```diff
@@ -150,103 +103,103 @@ function jsonToProjectComponent(json source) returns ProjectComponent {
 
 function jsonToIssue(json source) returns Issue {
     Issue target = {};
-    target.self = source.self.toString()?:"";
-    target.id = source.id.toString()?:"";
-    target.key = source.key.toString()?:"";
+    target.self = source.self.toString() ?: "";
+    target.id = source.id.toString() ?: "";
+    target.key = source.key.toString() ?: "";
 
     target.summary = source.fields != null ?
     source.fields.summary != null ?
-    source.fields.summary.toString()?:"" : "" : "";
+    source.fields.summary.toString() ?: "" : "" : "";
 
     target.creatorName = source.fields != null ?
     source.fields.creator != null ?
     source.fields.creator.name != null ?
-    source.fields.creator.name.toString()?:"" : "" : "" : "";
+    source.fields.creator.name.toString() ?: "" : "" : "" : "";
```

Comment:
Please check if we can use nil lifting concepts to avoid checking the nested fields individually here !
Eg:
```ballerina
target.creatorName = source.fields.creator.name.toString() ?: ""
```